python/TestHarness/util.py

```diff
@@ -6,7 +6,7 @@
 import json
 
 TERM_COLS = int(os.getenv('MOOSE_TERM_COLS', '110'))
-TERM_FORMAT = os.getenv('MOOSE_TERM_FORMAT', 'njcst')
+TERM_FORMAT = os.getenv('MOOSE_TERM_FORMAT', 'njCst')
```

Comment:
Not sure if we want this to be the default. Specifically, the "C" means that all caveats will be upper case. So messages like `Requires libMesh/libmesh#1409` will be output as `REQUIRES LIBMESH/LIBMESH#1409`
Maybe @permcody has an opinion.

Reply:
@brianmoose I agree!

The only reason I made it the default, is because unittests is expecting a lot of caveats to be upper-case.

I can change that; make it look pretty for us by default, and alter the environment when unittests are run so it passes unittests... but to me that sounds a little wrong (the default should be passing unittests).

Unless I am being overly protective of unittesting... I have no problems making this change if the rest of us are okay with it.

It would be completely transparent to everyone. `./run_tests` while in moose/python will still work no matter what you have set as your preferred MOOSE_TERM_FORMAT.